man/ocitools-generate.1.md

```diff
@@ -168,7 +168,9 @@ read the configuration from `config.json`.
   Values are "shared, rshared, private, rprivate, slave, rslave"
 
 **--rootfs**=ROOTFSPATH
-  Path to the rootfs
+  Path to the rootfs, which is relative to bundle path.
+  e.g the absolute path of rootfs is /to/bundle/rootfs, bundle path is /to/bundle,
+  then the value set as ROOTFSPATH should be `rootfs`.
```

Comment:
This is useful information, but I think we want to land it in runtime-spec first.  Currently [runtime-spec is fairly weak on how relative paths should be handled][1], although relative path semantics for `cgroupsPath` have [become more clear recently][2].

[1]: https://github.com/opencontainers/runtime-spec/issues/509#issuecomment-228414738
[2]: https://github.com/opencontainers/runtime-spec/pull/493/files#diff-428eec4013a655816cdefafd5d3505f1R164